gradle.properties

```diff
@@ -1,3 +1,3 @@
 package=co.com.bancolombia
-systemProp.version=2.2.3
```

Comment:
this line is change by a pipeline of github actions when the release is launched, the number is take from the tag